packages/common/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -1184,41 +1172,6 @@ export class AudiusAPIClient {
     return data.map(adapter.makeFavorite).filter(removeNullable)
   }
 
-  async getFavoritedTracks({
```

Comment:
I had really hoped more of apiClient wasn't used 😅 . But a lot of these are single calls in one place which can be replaced with sdk calls, so not a huge lift.